src/airtop/client.py

```diff
@@ -0,0 +1,90 @@
+from .base_client import BaseClient, AsyncBaseClient
+from .wrapper.windows_client import AirtopWindows, AsyncAirtopWindows
+from .wrapper.sessions_client import AirtopSessions, AsyncAirtopSessions
+
+class Airtop(BaseClient):
+    """
+    Use this class to access the different functions within the SDK. You can instantiate any number of clients with different configuration that will propagate to these functions.
+
+    Parameters
+    ----------
+    base_url : typing.Optional[str]
```

Comment:
Can we add "defaults to https://api.airtop.ai/api/v1/" here?

Reply:
@culli do you mean documentation wise?
This documentation is a copy of the documentation generated by fern.
I wonder if we should update the base_url documentation directly in huma.

We would have to update this documentation everytime code documentation changes though.

Reply:
@culli After reviewing the code, the client is using `base_url` or if no `base_url` provided, it will use one of the values of `AirtopEnvironment`.

So seems like `base_url` is a more flexible way to provide an alternative to the pre-defined environments.

Reply:
I don't see base_url in the openapi.yml, there is `servers: url: https://api.airtop.ai/api/v1`.   So I guess that's generated by Fern.  I think it's fine this way, as long as it's documented at some point.  The Node SDK I think just has the one param that is like `string | Environment`, but Python types work differently, so it is separated.